coordinator/main.go

```diff
@@ -90,7 +96,10 @@ func run() (retErr error) {
 				promhttp.HandlerOpts{Registry: promRegistry},
 			),
 		))
-		if err := http.ListenAndServe(":"+metricsPort, mux); err != nil {
+		metricsServer.Addr = ":" + metricsPort
+		metricsServer.Handler = mux
+		if err := metricsServer.ListenAndServe(); err != nil && !errors.Is(err, http.ErrServerClosed) {
+			logger.Error("serving prometheus /metrics endpoint", "err", err)
```

Comment:
```suggestion
			logger.Error("serving Prometheus /metrics endpoint", "err", err)
```
or align with the error below by doing both in lowercase